Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -14,6 +14,65 @@ following sections describe how to achieve that.
 A preview renderer is used to facilitate (record) previews in TYPO3. This
 class is responsible for generating the preview and the wrapping.
 
+The default PreviewRenderer is :php:`StandardContentPreviewRenderer` and handles
+the core's built-in CTypes.
+
+Extend the default PreviewRenderer
+==================================
+
+There's two ways of how you can provide previews for your own custom CTypes:
+via PageTS or event listener.
+
+PageTS
+------
+
+..  code-block:: typoscript
+
+    mod.web_layout.tt_content.preview.<CType> = EXT:my_extension/Resources/Private/Templates/Preview/ExampleCType.html
```

Comment:
Long lines of code cause horizontal scrolling in the frontend. I would suggest to use nesting at some level instead